deploy-cloudrun/src/service.ts

```diff
@@ -110,22 +131,55 @@ export class Service {
   }
 
   /**
-   * Parses a string of the format `KEY1=VALUE1`.
+   * Merges old revision with new service.
    *
-   * @param envVarInput Env var string to parse.
-   * @returns EnvVar[].
+   * @param prevService the previous Cloud Run service revision
    */
-  protected parseEnvVars(envVarInput: string): EnvVar[] {
-    const envVarList = envVarInput.split(',');
-    const envVars = envVarList.map((envVar) => {
-      if (!envVar.includes('=')) {
-        throw new TypeError(
-          `Env Vars must be in "KEY1=VALUE1,KEY2=VALUE2" format, received ${envVar}`,
-        );
+  public merge(prevService: run_v1.Schema$Service): void {
```

Comment:
This logic appends to or overwrites previous variables. If first set is `TEST_ENV=TEST_VAR,TEST_ENV2=TEST_VAR2` and second set is `TEST_ENV=NEW!` then the service would have `TEST_ENV=NEW!,TEST_ENV2=TEST_VAR2` (so it's always updating). If a user manual removes env vars it will redeploy that new version and this action will grab that last revision. Future enhancement will allow the users to choose update or set.